library/src/main/java/com/testproject/library/ScreenUtils.kt

```diff
@@ -0,0 +1,34 @@
+package com.testproject.library
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/errysuprayogi/ReviewMe/blob/8e177c6791b29597c6309ee6405dc2c695977e74/library/src/main/java/com/testproject/library/ScreenUtils.kt#L1"></span>The file /home/runner/work/ReviewMe/ReviewMe/library/src/main/java/com/testproject/library/ScreenUtils.kt is not ending with a new line.</td>
    </tr>
  </tbody>
</table>